trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Lift/vortex_lift.py

```diff
@@ -0,0 +1,48 @@
+## @ingroup Methods-Aerodynamics-Fidelity_Zero-Lift
```

Comment:
Why are you duplicating this file?

This file exists in fidelity zero supersonic. If you need it too move it to common